install/installer/pkg/components/database/cloudsql/deployment.go

```diff
@@ -43,7 +44,9 @@ func deployment(ctx *common.RenderContext) ([]runtime.Object, error) {
 						Labels:    labels,
 					},
 					Spec: corev1.PodSpec{
-						Affinity:                      &corev1.Affinity{},
+						Affinity: &corev1.Affinity{
```

Comment:
Does it make sense to extend the tests to check this is set correctly?

Reply:
Thx for finding this! Excellent idea, but I fear personally I won't time today, so opting for the "follow up" choice.